### ISSUE TITLE
Fuel consumption is perhaps a bit high

### ISSUE THREAD

**Issue by AndersHogqvist** [guifarias31/msfs_tbm930_project#4]
I'm by no means an expert on these things, but if I understand it correctly the picture below shows me flying at FL290, ISA+10 (approximately) and with 86% TRQ. According to the POH that would give me a fuel consumption at around 55.6 GPH but as you can see I'm getting a significantly higher value (69GPH). I know that the developers have been speaking about modeling the aircraft as one that has a few flight hours so perhaps that would give you a slightly higher fuel consumption, but not this much higher right?
![20200922_224010](https://user-images.githubusercontent.com/6253101/93935278-a4937a80-fd24-11ea-918a-470e06555d34.jpg)

**Comment by guifarias31** [comment]
I agree with you, it is, but nothing I do seems to make any change. I tried changing "PowerSpecificFuelConsumption" but I didn't notice any result. The FF is increasing while climbing, while it should decrease. I made a little table to write down the results:
![image](https://user-images.githubusercontent.com/70555403/93949945-6a23e080-fd18-11ea-9008-92c954770ae8.png)
I need help to fix that.

**Comment by Liscor** [comment]
Hm, hopefully you can get this fixed. Mostly doing long range in FSE where I need the fuel savings. Thank you for your work. Very nice so far.

**Comment by guifarias31** [comment]
I like long range in the TBM too. The two possible solutions at the moment are to refuel in flight, using the upper menu, and not to use the mod.

**Comment by PharaohSteve** [comment]
What about raising the fuel level and dropping the weight of the fuel to keep from making performance compromises?

**Comment by guifarias31** [comment]
I'm getting closer to something now, seems like only 1 value makes a difference, so I can't ajust it to altitude/temperature, but it's better than nothing.
![image](https://user-images.githubusercontent.com/70555403/94073843-8aae7200-fdce-11ea-8623-5f3a4a37183f.png)
And the preliminary results:
![image](https://user-images.githubusercontent.com/70555403/94088163-25667b00-fde6-11ea-901b-261c1af636bc.png)
The downside is that, just like in the default plane, the fuel flow is still increasing while climbing and the fuel flow on lower altitudes is pretty low.
I'll be releasing this in 0.4.11 shortly.

**Comment by guifarias31** [comment]
> What about raising the fuel level and dropping the weight of the fuel to keep from making performance compromises?

I don't know how to do this.

**Comment by withinboredom** [comment]
I think this happens in real life because of a single reason: the temperature of the fuel drops at altitude making the energy density increase, while at lower altitudes the temperature of the fuel increases which decreases the energy density. I see a lot of comments around the models making references to `fuel density is ~6.7lbs per US gallon` which is correct, [at room temperature](https://www.engineeringtoolbox.com/jet-fuel-temperature-density-petroleum-volume-correction-ASTM-D1250-gravity-d_1944.html). It's likely this isn't modelled in FS2020 either A. They didn't know about it. or B. They don't know why.

**Comment by withinboredom** [comment]
Yeah, I believe it just isn't modeled as this value didn't change since take-off (it should have changed by some number greater than 0). The best we could do is write a mod that adjusts how much fuel is in the tank using the OAT and some fancy math (actually, I bet that's pretty similar to real life, except the energy density changes too and thus the fuel flow). Shouldn't be too hard and I'll probably do it in the next few days if I manage to get some free time.

![image](https://user-images.githubusercontent.com/1883296/95003376-ff5b8a80-05de-11eb-9bd7-7ec6951f1e13.png)

**Comment by withinboredom** [comment]
https://forums.flightsimulator.com/t/someone-goofed-some-math/294734

Found the bug and reported on zendesk. 😮
